MapboxNavigation/ImageCache.swift

```diff
@@ -0,0 +1,143 @@
+import Foundation
+
+class ImageCache: BimodalImageCache {
+
+    let memoryCache: NSCache<NSString, UIImage>
+    let diskCacheURL: URL = {
+        let fileManager = FileManager.default
+        let basePath = fileManager.urls(for: .cachesDirectory, in: .userDomainMask).first!
+        return basePath.appendingPathComponent("com.mapbox.directions.downloadedImages")
```

Comment:
Nit: `directions` is somewhat misleading, perhaps we could use `Bundle.mapboxNavigation.bundleIdentifier+suffix`? ×2

Reply:
This is helpful, you've answered one of my tail work questions.